keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -236,11 +240,14 @@ Once the tests show that the mirrors are good, we can adapt the official documen
 
 #### Alpha -> Beta Graduation
 
+- [ ] [krel] creates and publishes deb and rpm packages of Kubernetes components
+- [ ] post-publish tests are written and run as part of the release process
+- [ ] nightly builds will be built and published on a daily basis using [krel] which will be improved to take over this task from [kubepkg] making use of [pre-existing periodic jobs] (https://github.com/kubernetes/test-infra/blob/97cb34fa9e2bfc4af35de3e561cb9fc5a1094da1/config/jobs/kubernetes/sig-release/kubernetes-builds.yaml#L120-L166)
+- [ ] documentation written checked to be complete and correct. 
 
-- [anago] [creates packages][pkg-gen-kep] and published those packages as part of the release process
-- post-publish tests are in places and run as part of the release process
-- nightlies will be build and published on a daily basis
-- documentation is in place
+
+[kubepgk]: https://github.com/kubernetes/release/tree/master/cmd/kubepkg 
+[krel]: https://github.com/kubernetes/release/blob/master/docs/krel/README.md 
```

Comment:
I think Beta graduation should likely include publicly publishing documentation updates to offer an alternative option to consume the community owned packages, and publishing packages using both the old and new processes.

This would provide us the opportunity to get end-user driven feedback prior to GA

Reply:
Ah OK, I did not know that this would bring about an End User process change resulting in a new procedure for packages published using this implementation I had assumed no process change for End Users @kubernetes/sig-release 

If there is a process change that should be called out for sure.

We already have in Beta -> GA Graduation 
>  documentation written checked to be complete and correct.